src/test/architecture/test_architecture.py

```diff
@@ -31,7 +31,8 @@
                                                NodeStruct, PathStruct,
                                                TimerCallbackStruct
                                                )
-from caret_analyze.exceptions import InvalidArgumentError, ItemNotFoundError
+from caret_analyze.exceptions \
+    import InvalidArgumentError, ItemNotFoundError, MultipleItemFoundError
```

Comment:
Suggenstion to improve readability.
```suggestion
from caret_analyze.exceptions import (InvalidArgumentError, ItemNotFoundError,
                                      MultipleItemFoundError)
```